tests/fixtures/inventories/software/01_computer_full_inventory_soft_and_os.json

```diff
@@ -0,0 +1,90 @@
+{
+    "action": "inventory",
+    "content": {
+        "accesslog": {
```

Comment:
`accesslog`, `antivirus` and `versionprovider` shoul be removed from all json tests files I guess (either they're not used or we will save time while running tests).